public/src/client/topic.js

```diff
@@ -143,7 +143,7 @@ define('forum/topic', [
 			if (components.get('post/anchor', postIndex).length) {
 				return navigator.scrollToPostIndex(postIndex, true);
 			}
-		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > 5) {
+		} else if (bookmark && (!config.usePagination || (config.usePagination && ajaxify.data.pagination.currentPage === 1)) && ajaxify.data.postcount > ajaxify.data.config.bookmarkThreshold) {
```

Comment:
Yes, he told me to use `ajaxify.data` instead of `config`. It seems like `config` doesn't necessarily get updated if the user changes preferences, but `ajaxify.data` always seems to be up to date.